### DISCUSSION TITLE
Problem with Android Manifest

### DISCUSSION THREAD

**Discussion by CloudFoxyVPN** [iBotPeaches/Apktool#3221] [Q&A]
![IMG_20230725_233720_414](https://github.com/iBotPeaches/Apktool/assets/133342273/d1d99788-7da1-45f4-83c6-0d04acf9ef65)

I'm having trouble configuring this permission to read my app's library in Android Manifest.

 android:extractNativeLibs="true"

 The problem is that when the app goes to Play Console review and then becomes available in the store...

 your manifest will be set to

 android:extractNativeLibs="false"

 this is done automatically, harming my application that depends on the libraries for its operation.

 When available locally in the case of manual installation outside the store, this permission is available as true normally.

 i am using the following command to get your aab

 < apktool >
 java -jar {apktool} d -m {apk_path} -s -o {decode_apk_dir} --copy-original"

 < Aapt2 >
   {aapt2} compile --legacy --dir {compile_source} -o {compiled_resources}

 < Aapt2 >
  {aapt2} link --proto-format -o {link_out_apk_path} -I {android} --min-sdk-version {min_sdk_version} --target-sdk-version {target_sdk_version} --version-code {version_code} --version -name {version_name} --manifest {input_manifest} --auto-add-overlay

 < Bundletool > Build-bundle >
 java -jar {bundletool} build-bundle --modules={modules} --output={out_aab_path} --config={bundle_config_json_path}

 there are other commands but I only mentioned the main codes that may be affecting extractNativeLibs to false which would be 0x0 instead of true 0x1.

 Has anyone had this problem and how can I solve it?

**Comment by iBotPeaches** [comment]
I was going to post that we have an existing issue around the `extractNativeLibs` value getting changed as a by-product of Apktool offering: https://github.com/iBotPeaches/Apktool/issues/2490

However, I get lost in some of your background.

When you say:

> When available locally in the case of manual installation outside the store, this permission is **available as true normally.**

Is that from your invoking aapt2 normally to compile an Apktool decoded application on host?
